### ISSUE TITLE
Unable to edit first, middle and last name separately

### ISSUE THREAD

**Issue by j-ed** [owncloud/contacts#830]
## Expected behavior
If you want to edit a name it should be possible to edit all name components, like first name, last name, title etc. as it was possible in oC v7.0.4

## Actual behavior
It is not possible to edit all name components like first name, last name, title etc. anymore, as it was possible in oC v7.0.4. The button behind the name field is missing.

![20150218_oc8_contact_name_details](https://cloud.githubusercontent.com/assets/2572764/6255309/726b8e32-b7af-11e4-937e-846d6a7a5a6e.jpg)

## Steps to reproduce
1.) Start the Contacts app
2.) Edit the name of an existing contact or add a new contact.
3.) It is not possible to edit the different parts of a name separately.

#### Server Configuration

OS: Linux 3.2.54
Web server: Apache2 2.4.10
Database: MySQL 5.1.73
PHP version: 5.4.35
ownCloud version: 8.0.0
Contacts app: v0.3.0.18

#### Client Configuration

Operating system: Windows 7
Browser: Mozilla Firefox v35.0.1

**Comment by m2u-germany** [comment]
Same here, tried Firefox 35 and Chrome 40.

**Comment by eckelmi** [comment]
Yes, that is exactly the problem I reported in #823 - I also described a quick solution of how to fix it (a line has to be added to the css file).

**Comment by j-ed** [comment]
I can confirm that the described fix solved the problem for me too.
